src/components/SecondaryPanes/index.js

```diff
@@ -30,6 +31,24 @@ const Accordion = React.createFactory(require("../shared/Accordion"));
 const CommandBar = React.createFactory(require("./CommandBar").default);
 require("./SecondaryPanes.css");
 
+type StartItems = {
```

Comment:
Hi @clarkbw 

I totally got that wrong. Sorry 😅. I'll push a fix right now. Thank you!